client/components/Navbar.tsx

```
@@ -109,97 +260,385 @@ const Navbar = () => {
+                  >
+                    {isLoadingResults ? (
+                      <>
+                        <Loader2 className="h-4 w-4 animate-spin mr-2" />
+                        Loading...
+                      </>
+                    ) : (
+                      "Load more"
+                    )}
+                  </button>
+                ) : (
+                  <div></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)